tests/data/application-data.js

```diff
@@ -0,0 +1,53 @@
+export const application = {
+  status: 'AGREED',
+  reference: 'IAHW-G3CL-V59P',
+  data: {
+    reference: 'TEMP-AAAA-AAAA',
+    declaration: true,
+    offerStatus: 'accepted',
+    confirmCheckDetails: 'yes'
+  },
+  organisation: {
+    name: 'Fake org name',
+    farmerName: 'Fake farmer name',
+    email: 'fake.farmer.email@example.com.test',
+    sbi: '123456789',
+    address: '1 fake street,fake town,United Kingdom',
+    orgEmail: 'fake.org.email@example.com.test'
+  },
+  createdAt: new Date(),
+  statusHistory: [
+    {
+      status: 'AGREED',
+      createdBy: 'admin',
+      createdAt: '2025-04-02T08:46:19.637Z'
+    }
+  ],
+  updateHistory: [],
+  contactHistory: [],
+  flags: [
+    {
+      id: '0b401d15-b594-4bce-851a-0f676f1ce5a6',
+      note: "User did not agree with multi herds T&C's",
+      deleted: true,
+      createdAt: '2025-04-30T10:42:04.707Z',
```

Comment:
Should we be inserting these as dates now then as per conversation yesterday where we agreed to go down that route instead of using strings?